tools/make-npm.sh

```diff
@@ -35,6 +35,9 @@ else
     mv $tarballname ../uBlock0.npm.tgz
 fi
 ln -sf "$TMPDIR/node_modules"
+if [ -z "$GITHUB_ACTIONS" ]; then
```

Comment:
For reference: https://docs.github.com/en/actions/reference/environment-variables